### PR TITLE
[Changelog]: Stable release for Email, Mailbox, Composer & Availability

### PR DESCRIPTION
**Pull request by pooja169usp**
# Code changes

- Updated changelog release version

# License

I confirm that this contribution is made under the terms of the MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: components/composer/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,4 @@
-# Pending Changes
```
Comment (krisgardiner) [inline components/composer/CHANGELOG.md:1]: Would you mind adding a blank template to the top of these changelog files for the next person?
```
# Pending Changes

## Breaking Changes
## New Features
## Bug Fixes
```

File: components/email/CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
-# v1.1.6 (2021-12-15)
```
Comment (krisgardiner) [inline components/email/CHANGELOG.md:1]: Same here.

File: components/mailbox/CHANGELOG.md [hunk 3]
```diff
@@ -1,12 +1,9 @@
-# v1.1.5 (2021-12-15)
```
Comment (krisgardiner) [inline components/mailbox/CHANGELOG.md:1]: Same here 🙂

**Review by krisgardiner — APPROVED** [review]

File: components/composer/CHANGELOG.md (hunk 1 above)
Comment (pooja169usp) [reply components/composer/CHANGELOG.md:1]: Done 👍🏽

**Pull request merged**